### PR TITLE
Complete isfrozen function term entry

### PR DESCRIPTION
**Pull request by 0umkbe**
### Description

Adding new entry regarding the `.isfrozen()` function in Luau language.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+-- Output the result
+if frozenStatus then
+    print("The table is frozen (read-only).")
+else
+    print("The table is not frozen (read/write).")
+end
+```
+
+In this example, we first create a table called myTable. Then, we use the isfrozen() function to check if it's in read-only mode. Depending on the result, we print a message indicating whether the table is frozen (read-only) or not.
```
Comment (SSwiniarski) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:50]:
```suggestion
In this example, a table called `myTable` is created. Then, the `isfrozen()` function is used to check if it's in read-only mode. Depending on the result, a message is printed indicating whether the table is frozen (read-only) or not.
```
After these edits, let's move this before the example code block.

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```luau
+isfrozen(table);
+```
+
+The 'isfrozen()' function returns a boolean value, true if the table is in read-only mode and false if it's not.
+
+
+
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:33]:
```suggestion
## Example
```
This isn't a codebyte. Codebytes are special code blocks that the reader can execute. It's only supported for some languages, and Luau isn't one of them.

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+```luau
+-- Create a table
+local myTable = {1, 2, 3}
+
+-- Check if the table is initially frozen (read-only)
+local frozenStatus = myTable:IsFrozen()
```
Comment (SSwiniarski) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:40]:
```suggestion
local frozenStatus = table.isfrozen(myTable)
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@0umkbe I did a first review and left a few comments.

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+
+---
+Title: '.isfrozen'
```
Comment (SSwiniarski) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:3]:
```suggestion
Title: 'isfrozen()'
```

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (SSwiniarski) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:1]:
```suggestion
```
Remove blank line at start of entry.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@0umkbe a couple more comments.

**Review by SSwiniarski — APPROVED** [review]
@0umkbe looks good for a second review.

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+The **`isfrozen()`** function in the Luau programming language returns a boolean based on whether the table is in read-only mode or not. When a table is in read-only mode, it means that the contents can't be modified, as a measure of data protection.
+
+## Syntax
+
+```pseudo
+isfrozen(table);
+```
+
+The 'isfrozen()' function returns a boolean value, true if the table is in read-only mode and false if it's not.
```
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:21]:
```suggestion
The `isfrozen()` function returns a boolean value, `true` if the table is in read-only mode and `false` if it's not.
```

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+## Example
+
+```luau
```
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:25]:
```suggestion
```lua
```

Comment (caupolicandiaz) [reply content/luau/concepts/tables/terms/isfrozen/isfrozen.md:25]: I don't believe luau is supported by markdown code blocks, but Lua can be used to show the syntax highlighting

File: content/luau/concepts/tables/terms/isfrozen/isfrozen.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+-- Output the result
+if frozenStatus then
+    print("The table is frozen (read-only).")
+else
+    print("The table is not frozen (read/write).")
+end
+```
+
+In this example, a table called `myTable` is created. Then, the `isfrozen()` function is used to check if it's in read-only mode. Depending on the result, a message is printed indicating whether the table is frozen (read-only) or not.
```
Comment (caupolicandiaz) [inline content/luau/concepts/tables/terms/isfrozen/isfrozen.md:40]:
```suggestion
In this example, a table called `myTable` is created. Then, the `isfrozen()` function is used to check if it's in read-only mode.

Comment (caupolicandiaz) [reply content/luau/concepts/tables/terms/isfrozen/isfrozen.md:40]: This is generally set before the example. And an output ```shell block is used to highlight the result

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@0umkbe I've added some additional comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by 0umkbe** [comment]
@caupolicandiaz Thanks for the feedback, please check now!

**Review by caupolicandiaz — APPROVED** [review]
@0umkbe @SSwiniarski There were some missing edits that I just made directly. This is good to merge

**Comment by SSwiniarski** [comment]
@caupolicandiaz @0umkbe we'll need to wait for the table entry for Luau is complete to merge this.

**Comment by 0umkbe** [comment]
@SSwiniarski Sure, looking forward to it!

**Pull request merged**